www/src/utils/reach-router-state-manager.js

```diff
@@ -1,33 +1,58 @@
-import React from "react"
+import React, { Component, Fragment } from "react"
 import queryString from "query-string"
 import { navigate } from "@reach/router"
 
-// manage your state entirely within the router, so that it's copiable
-// https://gist.github.com/sw-yx/efd9ee71669413bca6a895d87e30742f
+const emptySearchState = { s: ``, c: [], d: [], v: [], sort: `recent` }
```

Comment:
I don't think this is new in this PR, but why the single letter params here? It's harder to understand what's going on here!

Reply:
Yeah, this isn't new to the PR. I'll leave it for now, but I def see your point.